vibe_code_apps_22/eplus_native/align.py

```
@@ -119,24 +119,47 @@ def mae_rmse_mbe(y_true: np.ndarray, y_pred: np.ndarray) -> dict[str, float]:
     mae = float(np.mean(np.abs(err)))
     rmse = float(np.sqrt(np.mean(err**2)))
     mbe = float(np.mean(err))
-    mean_obs = float(np.mean(y_true))
-    nmbe = float(np.sum(y_true - y_pred) / (n * mean_obs) * 100.0) if abs(mean_obs) > 1e-12 else float("nan")
+    # Authoritative NMBE% from multires engine (m − ŷ, dof = n − p)
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `−` (MINUS SIGN). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)